File: src/main/java/com/example/ecommerce/entity/Address.java
package com.example.ecommerce.entity;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name="addresses")
public class Address {
    @Id
    @GeneratedValue
    private Long id;
    private String address;
    private String city;
    private String country;
    private String telephone;
}

File: web/src/app/common/address.ts
export class Address {

    street: string;
    city: string;
    state: string;
    country: string;
    zipCode: string;

    constructor(street: string,city: string,state: string,country: string,zipCode: string) {
        this.state = state;
        this.street = street;
        this.city = city;
        this.country = country;
        this.zipCode = zipCode;
    }
}
